app/styles/govflanders/components/_modal.scss

```diff
@@ -12,10 +12,18 @@ $modal-content-padding: 1.5rem;
   .vl-modal__backdrop {
     @include backdrop;
   }
+
+  .vl-modal__backdrop {
+    z-index: 52;
+  }
+
+  tbody {
+    background-color:white;
```

Comment:
Code style issue - put a space after the semicolon.

Reply:
I copied/refactored this code over from the main `app.scss`, but here the boyscout rule applies, so I will make the changes.